File: patterns/simple-state-machine/rules-js/simple-state-machine.js
rules.JSRule({
  name: "TimeOfDay MORNING (nur Wochentage)",
  triggers: [triggers.GenericCronTrigger("0 0 6 * * ?")],
  execute: (event) => {
    if (actions.Ephemeris.isWeekday()) {
      items.getItem("TimeOfDay").sendCommand("MORNING");
    }
  }
});

File: patterns/simple-state-machine/rules-python/simple_state_machine.py
from openhab import rule, Registry
from openhab.actions import Ephemeris
from openhab.triggers import GenericCronTrigger


@rule(triggers=[GenericCronTrigger("0 0 6 * * ?")])
class TimeOfDayMorningNurWochentage:
    def execute(self, module, input):
        if Ephemeris.isWeekday():
            Registry.getItem("TimeOfDay").sendCommand("MORNING")
